syntax = "proto3";

package example;


// Message sent to increase the count of the user
message IncreaseUserCount{
    string name = 1;
}

// State of the user
message User{
    string name = 1;
    int64 count = 2;
}

message ExampleResponse{
    string message = 1;
}

message ExampleRequest{
    string message = 1;
}
